### PR TITLE
Added imagePullSecrets to admission-controller and proxy-scanner. Version increased to 0.24.0

### PR DESCRIPTION
**Pull request by fabnord**
This PR adds optional support for imagePullSecrets to the admission-controller and proxy-scanner helm charts. This is required for prospects and customers using their own container registries. 

In addition the version numbers have been increase from 0.23.0 to 0.24.0 for the helm-charts and images used. It needs to be clarified if this works as part of the release process of the container images for admission-controller and proxy-scanner.

### CODE REVIEW

File: admission-controller/Chart.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-version: 0.23.0
+version: 0.24.0
 name: admission-controller
 description: Lacework admission controller using Lacework proxy scanner
 type: application
 keywords:
-- admission-controller
-- runtime-security
-- lacework
+  - admission-controller
```
Comment (nschmeller) [inline admission-controller/Chart.yaml:8]: Are these changes needed?

File: admission-controller/values.yaml [hunk 2]
```diff
@@ -12,12 +12,12 @@ admission:
   # All other resources may spawn multiple child resources,
   # and you will need to exclude all of these if you want to prevent excessive scanning during the deployment.
   excluded_resources:
-  - "Deployment"
-  - "ReplicaSet"
-  - "DaemonSet"
-  - "Job"
-  - "CronJob"
-  - "StatefulSet"
+    - "Deployment"
```
Comment (nschmeller) [inline admission-controller/values.yaml:15]: Are these changes needed?

File: admission-controller/values.yaml [hunk 3]
```diff
@@ -63,7 +63,8 @@ securityContext:
   fsGroup: 11433
 container_securityContext: {}
 
-resources: {}
+resources:
```
Comment (nschmeller) [inline admission-controller/values.yaml:66]: Are these changes needed?

File: admission-controller/values.yaml [hunk 4]
```diff
@@ -75,8 +76,12 @@ nodeSelector: {}
 
 deployment:
   image: lacework/lacework-admission-controller
-  tag: 0.23.0
+  tag: 0.24.0
   pullPolicy: Always
+  # [Optional] imagePullSecrets.
+  # https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  # imagePullSecrets:
```
Comment (nschmeller) [inline admission-controller/values.yaml:83]: Should this be commented out?

File: admission-controller/values.yaml [hunk 5]
```diff
@@ -98,25 +112,25 @@ proxy-scanner:
     lacework:
       account_name:
       integration_access_token:
-#    registry_secret_name:
+    #    registry_secret_name:
     registries:
-#      - domain:
```
Comment (nschmeller) [inline admission-controller/values.yaml:103]: Are these changes needed?

**Review by nschmeller — COMMENTED** [review]
Hi @fabnord, thanks for the change request! On my first pass, it looks like there are a lot of style changes that should be separate from this PR. Could you remove them or explain the motivation for adding them? Additionally, please sign the CLA if you haven't already

**Comment by fabnord** [comment]
Closing PR as changes happen outside of this repo.
